modules/apps/site-initializer/site-initializer-testray/site-initializer-testray/extra/remote-app/src/pages/Project/Cases/CaseResultHistory.tsx

```diff
@@ -28,85 +31,112 @@ type CaseResultHistoryProps = {
-			},
-		}}
-		managementToolbarProps={{
-			title: i18n.translate('test-history'),
-			visible: true,
-		}}
-		resource={testrayCaseResultImpl.resource}
-		tableProps={{
-			columns: [
-				{
-					clickable: true,
-					key: 'dateCreated',
-					render: (date) => dayjs(date).format('lll'),
-					size: 'sm',
-					value: i18n.translate('create-date'),
+}) => {
+	const {caseResultId} = useParams();
+
+	return (
+		<ListView
+			initialContext={{
+				columns: {
+					caseType: false,
+					dateCreated: true,
+					dateModified: false,
+					issues: false,
+					team: false,
 				},
-				{
-					clickable: true,
-					key: 'build',
-					render: (build) => {
-						return build?.gitHash;
+			}}
+			managementToolbarProps={{
+				title: i18n.translate('test-history'),
+				visible: true,
+			}}
+			resource={testrayCaseResultImpl.resource}
+			tableProps={{
+				columns: [
+					{
+						clickable: true,
+						key: 'dateCreated',
+						render: (date) => (
+							<p style={{maxWidth: '11ch'}}>
+								{' '}
+								{dayjs(date).format('lll')}
+							</p>
+						),
+						size: 'sm',
+						value: i18n.translate('create-date'),
 					},
-					value: i18n.translate('git-hash'),
-				},
-				{
-					clickable: true,
-					key: 'product-version',
-					render: (_, {build}) => build?.productVersion?.name,
-					value: i18n.translate('product-version'),
-				},
-				{
-					clickable: true,
-					key: 'run',
-					render: (run) => run?.externalReferencePK,
-					size: 'lg',
-					value: i18n.translate('environment'),
-				},
-				{
-					clickable: true,
-					key: 'routine',
-					render: (_, {build}) => build?.routine?.name,
-					value: i18n.translate('routine'),
-				},
-				{
-					key: 'dueStatus',
-					render: (dueStatus: PickList) => {
-						return (
-							<StatusBadge
-								type={dueStatus.key as StatusBadgeType}
-							>
-								{dueStatus.name}
-							</StatusBadge>
-						);
+					{
+						clickable: true,
+						key: 'build',
+						render: (build) => {
+							return build?.gitHash;
+						},
+						value: i18n.translate('git-hash'),
 					},
-					value: i18n.translate('status'),
-				},
-				{
-					key: 'warnings',
-					value: i18n.translate('warnings'),
+					{
+						clickable: true,
+						key: 'product-version',
+						render: (_, {build}) => build?.productVersion?.name,
+						thunkableTitle: true,
+						value: i18n.translate('product-version'),
+					},
+					{
+						clickable: true,
+						key: 'run',
+						render: (run) => run?.externalReferencePK,
+						value: i18n.translate('environment'),
+					},
+					{
+						clickable: true,
+						key: 'routine',
+						render: (_, {build}) => build?.routine?.name,
+						value: i18n.translate('routine'),
+					},
+					{
+						key: 'dueStatus',
+						render: (dueStatus: PickList) => {
+							return (
+								<StatusBadge
+									type={dueStatus.key as StatusBadgeType}
+								>
+									{dueStatus.name}
+								</StatusBadge>
+							);
+						},
```

Comment:
Simplify returning directly
```suggestion
						render: (dueStatus: PickList) =>  (
								<StatusBadge
									type={dueStatus.key as StatusBadgeType}
								>
									{dueStatus.name}
								</StatusBadge>
							),
```